.github/CODEOWNERS

```diff
@@ -8,5 +8,5 @@
 * @googleapis/ruby-team
 
 # Autogeneration PRs should be handled by yoshi-approver
-/generated/ @yoshi-approver
+/generated/ @yoshi-approver @googleapis/ruby-team
 /api_names_out.yaml @yoshi-approver
```

Comment:
Add  it to `api_names_out.yaml` as well, as it can also be modified when libraries are regenerated.

Reply:
Done